tests/features/rhdm/rhdm-kieserver.feature

```diff
@@ -31,3 +31,19 @@ Feature: RHDM KIE Server configuration tests
   Scenario: Check rhdm-kieserver extensions
     When container is ready
     Then container log should contain -Dorg.jbpm.server.ext.disabled=true -Dorg.jbpm.ui.server.ext.disabled=true -Dorg.jbpm.case.server.ext.disabled=true
+
+  @wip
+  Scenario: Check JBPM_CLUSTER flag enabled
+    When container is started with env
+      | variable                        | value                |
+      | JGROUPS_PING_PROTOCOL           | kubernetes.KUBE_PING |
+      | JBPM_CLUSTER                    | true                 |
+    Then container log should contain JBPM_CLUSTER enabled
```

Comment:
also check if the standalone-openshift.xml contains the configuration you are applying in the `configure_jbpm_ha`